#162474707 implement all landing page functionality

ucheg6 opened the pull request:
#### What does this PR do?
implement all landing page functionality 
#### Description of Task to be completed?
- display author details where necessary in the landing page
- connect link to read single articles component
- connect link to creat article component
- create bookmark article saga and dispatch action to bookmark and unbookmark article

#### How should this be manually tested?
On command line ;
- git clone this repo
- npm install 
- Run `npm start:dev ` from the `haven-ah-frontend` directory

#### What are the relevant pivotal tracker stories?
[#162474707](https://www.pivotaltracker.com/story/show/162474707)

#### Screenshots (if appropriate)>
NONE

TheoOkafor reviewed (commented):
This looks good @ucheg6

seyi-adeleke commented on public/index.html:
```diff
@@ -5,7 +5,7 @@
         <meta http-equiv="X-UA-Compatible" content="ie=edge">
         <title>Authors Haven</title>
-        <base href="/"/>
+        <base href="/" />
```
why this space?

ucheg6 replied:
I've removed it. Thanks

seyi-adeleke commented on public/styles/App.scss:
```diff
@@ -253,4 +253,5 @@ h1, h2, h3, h4, h5, h6 {
-}
\ No newline at end of file
+}
+a { color: inherit; } 
```
Are you using this?

ucheg6 replied:
Yeah I am

seyi-adeleke commented on src/actions/bookmarkArticle.js:
```diff
@@ -0,0 +1,34 @@
+} from '../actionTypes/bookmarkArticleActionTypes';
+
+export const bookmarkArticleAction = slug => ({
+  type: BOOKMARK_ARTICLE_REQUEST,
```
I thought we agreed to have consistent actions? `type` and `payload `?

ucheg6 replied:
Yeah. sorry about that. That has been implemented.

ucheg6 committed:
feat(landing-page): implement all functionalities in the landing page

display author details where necessary
connect to read single article component
connect to create article component
implement bookmark and unbookmark functionality
test all functionalities
